packages/terra-framework-docs/src/terra-dev-site/test/data-grid/flowsheet-data-grid/DefaultFlowsheetDataGrid.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { FlowsheetDataGrid } from 'terra-data-grid';
+
+const DefaultFlowsheetDataGrid = () => {
+  const gridDataJSON = {
+    cols: [
+      { id: 'Column-0', displayName: 'Vitals', isSelectable: false },
+      { id: 'Column-1', displayName: 'March 16' },
+      { id: 'Column-2', displayName: 'March 17' },
+      { id: 'Column-3', displayName: 'March 18' },
+    ],
+    rows: [
+      {
+        id: '1',
+        cells: [
+          { content: 'Heart Rate Monitored (bpm)' },
+          { content: '68' },
+          { content: '66' },
+          { content: '67' },
+        ],
+      },
+      {
+        id: '2',
+        cells: [
+          { content: 'Temperature Oral (degC)' },
+          { content: '36.7', isMasked: true },
+          { content: '36.9', isMasked: true },
+          { content: '37', isMasked: true },
+        ],
+      },
+      {
+        id: '3',
+        cells: [
+          { content: 'Cardiac Index (L/min/m2)' },
+          { content: '2.25' },
+          { content: '2.28' },
+          { content: '2.8' },
+        ],
+      },
+      {
+        id: '4',
+        cells: [
+          { content: 'ICP (mmHg)', isMasked: true },
```

Comment:
Note: Probably fine for a test but when we get to doc site examples, those should not include masked row headers.

Reply:
Yes, this was a carryover from the other data grids. I can exclude this for the consumer-facing examples